languages/csharp/exercises/concept/arrays/.meta/Example.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+
+public class BirdCount
+{
+    private int[] birdsPerDay;
+
+    public BirdCount(int[] birdsPerDay)
+    {
+        this.birdsPerDay = birdsPerDay;
+    }
+
+    public int Total()
+    {
+        var total = 0;
+
+        foreach (var count in birdsPerDay)
+        {
+            total += count;
+        }
+
+        return total;
+    }
+
+    public int BusyDays()
+    {
+        var days = 0;
+
+        foreach (var count in birdsPerDay)
+        {
+            if (count >= 5)
+            {
+                days++;
+            }
+        }
+
+        return days;
+    }
+
+    public int Yesterday()
+    {
+        return birdsPerDay[5];
+    }
+
+    public bool HasDayWithoutBirds()
+    {
+        return Array.IndexOf(birdsPerDay, 0) != -1;
+    }
+
+    public static int[] LastWeek()
+    {
+        return new int[] { 0, 2, 5, 3, 7, 8, 4 };
```

Comment:
It'd be better to have it stored somewhere (as the text states) instead of generated. Even if I think the compiler is probably smarter then us... it contradicts the text.

Reply:
Normally I'd use a `static readonly` field, but students won't know about that at this point (see [my previous comment](https://github.com/exercism/v3/pull/1257#discussion_r412737290)).